QSPIFBlockDevice.cpp

```diff
@@ -433,7 +406,7 @@ int QSPIFBlockDevice::erase(bd_addr_t addr, bd_size_t inSize)
         }
         wait_ms(QSPIF_DEFAULT_TIMEOUT_MSEC);
         if ( false == _is_mem_ready()) {
-            QSPIF_LOG(QSPIF_DEBUG_ERROR, "ERROR: QSPI After Erase Device not ready - failed\n");
+            tr_error("ERROR: QSPI After Erase Device not ready - failed\n");
             erase_failed = true;
             status = QSPIF_BD_ERROR_READY_FAILED;
             goto Exit_Point;
```

Comment:
@0xc0170 do we have a naming convention for labels? I would expect them to be similar to functions.

Reply:
not specifically but should be similar

Reply:
I'll fix to match convention